docs/diagrams/ParserClasses.puml

```diff
@@ -4,35 +4,35 @@ skinparam arrowThickness 1.1
 skinparam arrowColor LOGIC_COLOR_T4
 skinparam classBackgroundColor LOGIC_COLOR
 
-Class "{abstract}\nCommand" as Command
+Class "{abstract}\nAnimalCommand" as AnimalCommand
 Class XYZCommand
 
 package "Parser classes"{
-Class "<<interface>>\nParser" as Parser
-Class AddressBookParser
+Class "<<interface>>\nAnimalParser" as AnimalParser
+Class AnimalCatalogParser
 Class XYZCommandParser
 Class CliSyntax
```

Comment:
```suggestion
Class CliSyntax
```
Maybe you could change it to CliAnimalSyntax instead based on the naming in our tP.